lib/cron.js

```diff
@@ -267,9 +270,15 @@
 					continue;
 				}
 
+				if(date.valueOf()==start){
+					date.seconds(date.seconds()+1);
+				}
+
 				break;
 			}
-
+			if(origDate.valueOf()==start){
```

Comment:
Why would we want to go back a second here?

Reply:
That is to remove the effect of adding 1 second as shown in line 274.
The reason why i' am adding 1 second is to cover cases were all the "if" statements don't match; that would usually occur when startDate is Also one of the previous Dates.
example of reproducing that case can be found here:
https://repl.it/repls/DarkcyanNiceConcentrate

Reply:
@ncb000gt  removed in  commit `98dad71`